lib/charms/opensearch/v0/opensearch_tls.py

```diff
@@ -73,7 +73,7 @@ def __init__(
         self.jdk_path = jdk_path
         self.certs_path = certs_path
         self.keytool = self.jdk_path + "/bin/keytool"
-        self.certs = TLSCertificatesRequiresV3(charm, TLS_RELATION)
+        self.certs = TLSCertificatesRequiresV3(charm, TLS_RELATION, expiry_notification_time=168)
```

Comment:
Tiny nitpick, can B ignored.

May it be useful to add a comment saying 
```
We want to be notified a week in advance
```
or so (highlighting the uing of 168 :-) )